middleware/idempotency/idempotency_test.go

```diff
@@ -83,7 +83,7 @@ func Test_Idempotency(t *testing.T) {
 			req.Header.Set("X-Idempotency-Key", idempotencyKey)
 		}
 		resp, err := app.Test(req, fiber.TestConfig{
-			Timeout:       15 * time.Second,
+			Timeout:       115 * time.Second,
```

Comment:
Hmmm

Reply:
oh, sorry haha. Now this commit is not included btw